### ISSUE TITLE
Crash with high dimensional environments (e.g. "Pong-v0")

### ISSUE THREAD

**Issue by SebGruber1996** [smilesun/rlR#19]
Crash happens when executing these lines:
-------------------------------------------------------

env = makeGymEnv("Pong-v0") # also tried other high dimensional environments and they crashed, too
agent = makeAgent("AgentDQN", env)
perf = agent$learn(1) # here happens the crash

-------------------------------------------------------
Error message:
-------------------------------------------------------

Error in parse(text = text) : <text>:1:35: unexpected symbol
1: model <- keras_model_sequential() model
                                      ^

-------------------------------------------------------

How to fix:
- open file "nnArsenal.R"
- delete code lines 70 - 92
- insert the following code instead:

-------------------------------------------------------

  model = {
    opt <- optimizer_rmsprop(lr = 0.0001, decay = 1e-6)
    keras_model_sequential() %>%
      layer_conv_2d(filter = 32, kernel_size = c(3,3), padding = "same", input_shape = input_shape) %>%
      layer_activation("relu") %>%
      layer_conv_2d(filter = 32, kernel_size = c(3,3)) %>%
      layer_activation("relu") %>%
      layer_max_pooling_2d(pool_size = c(2,2)) %>%
      layer_dropout(0.25) %>%
      layer_conv_2d(filter = 32, kernel_size = c(3,3), padding = "same") %>%
      layer_activation("relu") %>%
      layer_conv_2d(filter = 32, kernel_size = c(3,3)) %>%
      layer_activation("relu") %>%
      layer_max_pooling_2d(pool_size = c(2,2)) %>%
      layer_dropout(0.25) %>%
      layer_flatten() %>%
      layer_dense(512) %>%
      layer_activation("relu") %>%
      layer_dropout(0.5) %>%
      layer_dense(act_cnt) %>%
      layer_activation("softmax") %>%
      compile(loss = "categorical_crossentropy", optimizer = opt, metrics = "accuracy")
  }

-------------------------------------------------------

Doing this fixed the error for me. It seems while parsing the strings together, some ";" were missing.
My solution doesn't require parsing, so replacing the whole code makes sense.

Greetings,
Sebastian

**Comment by smilesun** [comment]
fixed in master, I us parsing version because we want to be flexibible about the cnn structure.
